install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
So, there's two issues:
 - `nvm_bash_completion` should only touch *one* profile file, the return of `nvm_detect_profile`
 - I'm unconvinced if `nvm_bash_completion` should even happen automatically, instead of with an opt-in command line argument